scripts/create-kind-cluster-with-SSL-passthrough.sh

```diff
@@ -65,6 +71,27 @@ while (( $# > 0 )); do
   shift
 done
 
+
+###############################################################################
+# Prefetch the images required by nginx ingress
+###############################################################################
+prefetch_images=()
+prefetch_pids=()
+if [[ "$prefetch" == "true" ]] ; then
+  curl -s "$NGINX_INGRESS_URL" | sed 's/@sha256.*$//' > "$NGINX_INGRESS_FILE"
```

Comment:
Here is a way to suppress the progress meter without suppressing error messages:

```console
bash-3.2$ curl --no-progress-meter "$NGINX_INGRESS_URL" | sed 's/@sha256.*$//' > "$NGINX_INGRESS_FILE"
bash-3.2$ 
```